### PR TITLE
Show refresh period on Status page and Upgrade libraries

### LINKED ISSUE #100: Zookkeeper data reload improvements and docs
Recently we implemented periodic checks on zookeeper, see https://github.com/diennea/bookkeeper-visual-manager/pull/98

We should edit the "system" page to clearly state that, by default, there is a periodic reload enabled.
We should also add a properties to disable the "RELOAD NOW" button (in a theoretically environment with many users it could cause a DOS to zk: in these instances the administrator may want to disable the button and rely only on the periodic reloads)

### PR DESCRIPTION
**Pull request by mino181295**
### Improvement

- Upgrade libraries in the node project and upgrade java dependencies.
- Fix bug in UI in ledger age
- Resolve #100 and show refresh period on Status page

### CODE REVIEW

File: web/src/main/bvmui/src/components/Ledger.vue [hunk 1]
```diff
@@ -13,14 +13,25 @@
...
         </div>
     </div>
 </template>
 <script>
 export default {
     props: {
         ledger: Object
+    },
+    computed: {
+        computedAge() {
```
Comment (nicoloboschi) [inline web/src/main/bvmui/src/components/Ledger.vue:26]: Can we add some unit tests for this function?

**Review by nicoloboschi — COMMENTED** [review]
LGTM, I think it is better to add unit tests for the new time formatting function

File: pom.xml [hunk 2]
```diff
@@ -48,18 +48,18 @@
         <netty.version>4.1.51.Final</netty.version>
         <netty.boringssl.version>2.0.34.Final</netty.boringssl.version>
         <junit.version>4.13.1</junit.version>
-        <mockito.version>1.10.19</mockito.version>
+        <mockito.version>2.0.2-beta</mockito.version>
```
Comment (eolivelli) [inline pom.xml:51]: why do we need to update to a bet version ?
is there a real benefit ?
otherwise I am -1 here

**Review by eolivelli — CHANGES_REQUESTED** [review]
looks good a part from the mockito upgrade

File: web/src/main/bvmui/src/components/Ledger.vue (hunk 1 above)
Comment (mino181295) [reply web/src/main/bvmui/src/components/Ledger.vue:26]: Done, started the tests in the frontend

File: pom.xml (hunk 2 above)
Comment (mino181295) [reply pom.xml]: Okay, I downgraded to the latest stable

**Review by eolivelli — APPROVED** [review]
LGTM
Generally I don't like patches that touch too many things, in this case we are:
- upgrading test java libraries
- upgrading maven plugins
- fixing the UI
- adding a new little feature on the UI

**Pull request merged**